dev/com.ibm.websphere.appserver.features/visibility/public/mpFaultTolerance-4.1/io.openliberty.mpFaultTolerance-4.1.feature

```diff
@@ -0,0 +1,26 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpFaultTolerance-4.1
+visibility=public
+singleton=true
+IBM-App-ForceRestart: install, \
+ uninstall
+IBM-API-Package: org.eclipse.microprofile.faulttolerance.exceptions;  type="stable", \
+                 org.eclipse.microprofile.faulttolerance;  type="stable"
+IBM-ShortName: mpFaultTolerance-4.1
+Subsystem-Name: MicroProfile Fault Tolerance 4.1
+# io.openliberty.mpCompatible-5.0; ibm.tolerates:="6.0,6.1" comes from io.openliberty.mpConfig features
+-features=io.openliberty.mpConfig-3.0; ibm.tolerates:="3.1", \
```

Comment:
this should probably be only 3.1

Reply:
I agree that the feature file is not correct. In order to move forwards, I've opened #28463 to make those changes after this PR is merged.